lib/slack/lookups.ex

```diff
@@ -1,57 +1,43 @@
 defmodule Slack.Lookups do
+  
+  alias Slack.Client
+  
   @doc ~S"""
   Turns a string like `"@USER_NAME"` into the ID that Slack understands (`"U…"`).
   """
-  def lookup_user_id("@" <> user_name, slack) do
```

Comment:
AFAIK `Client` will never need to look these things up, it just ensures the state that lookups refer to stays up to date. When you `use Slack` you get both anyways. Agree with moving lookup logic.